ikarus/utils/functionhelper.hh

```diff
@@ -37,5 +39,23 @@ void obtainLagrangeNodePositions(const LV& localView,
   for (auto& nCoord : lagrangeNodeCoords)
     nCoord = localView.element().geometry().global(nCoord);
 }
+/**
+ * \brief A function to obtain the local coordinates of subentities of an FiniteElement
+ * \ingroup utils
+ * \tparam FE Type of the finite element
+ * \param fe finite element
+ * \param codim codim of requested subentity, defaults to vertices
+ * \return view over the position of the subenties
+ */
+template <typename FE>
+auto referenceElementSubEntitiyPositions(FE& fe, const int codim = FE::Traits::mydim) {
```

Comment:
The const in the function parameter is also not nice style. It should by just int